frontend/src/index.tsx

```diff
@@ -33,6 +34,10 @@ createRoot(document.querySelector('#root') as HTMLElement).render(
                 path: AppRoute.SIGN_UP,
                 element: <Auth />,
               },
+              {
```

Comment:
let's move RouterProvider out of index, to some separate component

Reply:
create new separate component Router and add in index.tsx